app/templates/gentelella/admin/mytickets/mytickets_list.html

```diff
@@ -26,7 +26,7 @@ <h4 class="name">{{ order.event.name }}</h4>
                     <hr>
                     <p class="tags">
                     <span class="filter-hashtag">
-                        {% if order.amount == 0 %}Free{% else %}${{ order.amount | money }}{% endif %}
+                        {% if order.amount == 0 %}Free{% else %} {% if order.event.payment_currency %} {{ order.event.payment_currency | currency_symbol }} {% else %} $ {% endif %} {{ order.amount | money }}{% endif %}
```

Comment:
The `payment_currency` in an event is a required field and is by default set to USD. So I think this `if` check is not necessary.